capsules/src/ccs811.rs

```diff
@@ -0,0 +1,330 @@
+//! SyscallDriver for the AMS CCS811 an ultra-low power digital gas sensor
+//! solution which integrates a metal oxide (MOX) gas sensor to detect a wide
+//! range of Volatile Organic Compounds (VOCs) for indoor air quality
+//! monitoring using the I2C bus.
+//!
+//! <https://cdn.sparkfun.com/assets/learn_tutorials/1/4/3/CCS811_Datasheet-DS000459.pdf>
+//!
+
+use core::cell::Cell;
+use kernel::dynamic_deferred_call::{
+    DeferredCallHandle, DynamicDeferredCall, DynamicDeferredCallClient,
+};
+use kernel::hil::i2c::{self, I2CClient, I2CDevice};
+use kernel::hil::sensors::{AirQualityClient, AirQualityDriver};
+use kernel::utilities::cells::{OptionalCell, TakeCell};
+use kernel::ErrorCode;
+
+const STATUS: u8 = 0x00;
+const MEAS_MODE: u8 = 0x01;
+const ALG_RESULT_DATA: u8 = 0x02;
+#[allow(dead_code)]
+const RAW_DATA: u8 = 0x03;
+#[allow(dead_code)]
+const ENV_DATA: u8 = 0x05;
+#[allow(dead_code)]
+const NTC: u8 = 0x06;
+#[allow(dead_code)]
+const THRESHOLDS: u8 = 0x10;
+#[allow(dead_code)]
+const BASELINE: u8 = 0x11;
+const HW_ID: u8 = 0x20;
```

Comment:
should this be `0x81` and used in line 219?

Reply:
This is the register address, `0x81` is the expected value at the address